scripts/manage_user.py

```
@@ -38,19 +38,21 @@ def reset_password(username: str, new_password: str):
         db.close()
 
 
-def create_user(username: str, password: str):
+def create_user(username: str, password: str, roles: list = None):
```

Ruff (0.15.9)

[warning] PEP 484 prohibits implicit `Optional`

Convert to `Optional[T]`

(RUF013)